packages/tx/src/eip1559Transaction.ts

```diff
@@ -196,6 +196,9 @@ export default class FeeMarketEIP1559Transaction extends BaseTransaction<FeeMark
    * @param baseFee The base fee of the block
    */
   getUpfrontCost(baseFee?: BN): BN {
+    if (!baseFee) {
+      baseFee = new BN(0)
```

Comment:
Could have defaulted this in the method argument.

Reply:
good idea, i've added it to [Tiny Fixes Meta Issue](https://github.com/ethereumjs/ethereumjs-monorepo/issues/905)

Reply:
also on the two lines underneath, baseFee doesn't need the non-null assertion operator (`!`)